RELEASING.md

```diff
@@ -1,20 +1,19 @@
-# Releasing a new version
+# Release guide
 
-## 1) Update the image tags in `master`
+## Branching/tagging scheme
 
-In the latest `master` branch:
+Just like deploy-sourcegraph, we use version branches and version tags. We _additionally_ have a second set which is customer-replication branches and version tags:
 
-1. Find and replace `:insiders` with `:3.0.0` on all files except this one. This is required for the next step.
-2. Run these commands:
+- Tag examples: `v3.8.2`, `v3.9.2`, `customer-replica-v3.8.2`, `customer-replica-v3.9.2-2`
+- Branch examples: `3.8`, `3.9`, `3.8-customer-replica`, `3.9-customer-replica`
 
-```
-go get -u github.com/slimsag/update-docker-tags
-update-docker-tags --constraint 'sourcegraph/prometheus=<10.0' --constraint 'sourcegraph/grafana=<6.0' .
-```
+The customer replica ones are important as we must maintain some diffs for replication purposes, such as using a different Postgres version, changes to the `prometheus_targets.yml` and more. We use tags to ensure each AMI we save correlates directly with an immutable Git tag.
```

Comment:
```suggestion
The customer replica branches are distinct living branches which have diffs we maintain for replicating some customers using the pure-docker deployment model described in the README.md of this repository (e.g. the customers' different Postgres version, changes to the `prometheus_targets.yml`, and more.) We test, tag, and release these `$VERSION-customer-replica` branches as `customer-replica-$VERSION` to produce the final diffs we send to customers running a pure-docker deployment on e.g. https://docs.sourcegraph.com/admin/updates/pure_docker
```

Rephrased to include some more detail, but also the AMI stuff won't be true soon because it's all outdated and I plan to delete them all.